modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -293,7 +425,7 @@ sealed trait HasChildQuery[S]
    * @param value
    *   the [[scala.Int]] value for `score` parameter
    * @return
-   *   a new instance of the [[HasChildQuery]] with the `score` value set.
+   *   an instance of [[HasChildQuery]] with the `score` value set.
```

Comment:
Maybe to follow same syntax, and write ...enriched with the `score` parameter? (if decided, do the same bellow)